PreFreeSurfer/scripts/ACPCAlignment.sh

```diff
@@ -8,65 +8,47 @@
 #  Usage Description Function
 # ------------------------------------------------------------------------------
 
-script_name=$(basename "${0}")
 
-Usage() {
-	cat <<EOF
+set -eu
 
-${script_name}: Tool for creating a 6 DOF alignment of the AC, ACPC line and hemispheric plane in MNI space
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    #fix this if the script is more than one level below HCPPIPEDIR
+    export HCPPIPEDIR="$(dirname -- "$0")/.."
```

Comment:
For all the scripts in a `scripts/` subdirectory, this should be `"$(dirname -- "$0")/../.."`.  If this line is set properly, then in a terminal with HCPPIPEDIR not set, running it without arguments will still produce the usage information.